eng/common/TestResources/build-test-resource-config.yml

```diff
@@ -14,6 +14,24 @@ parameters:
     default: null
 
 steps:
+  - task: AzurePowerShell@5
```

Comment:
Should we make this conditional if folks want to opt-out? Or is there a place where someone can override PoolSubnet?